utilities/message_filters/src/message_filters/__init__.py

```diff
@@ -99,18 +102,26 @@ def __init__(self, f, cache_size = 1):
         # Array to store msgs times, auxiliary structure to facilitate
         # sorted insertion
         self.cache_times = []
+        # Whether to allow storing headerless messages with current ROS
+        # time instead of timestamp.
+        self._allow_headerless = allow_headerless
```

Comment:
I think it would be ok to make the property "public" so that users can check the property later. So I would propose to remove the leading underscore.